supervisor/docker/manager.py

```diff
@@ -311,6 +311,7 @@ def run_command(
             container = self.docker.containers.run(
                 f"{image}:{tag}",
                 command=command,
+                detach=True,
```

Comment:
The change generally looks good, from what I understand this is required because we call `container.wait()` below.

That said, is there a good reason we go down that road? We could also run it attached and simply let the `container.run()` block? :thinking:

Reply:
> That said, is there a good reason we go down that road? We could also run it attached and simply let the `container.run()` block? 🤔

Indeed `container.run()` could be used and in fact I was using it in https://github.com/home-assistant/supervisor/pull/5974 before I found out this function.

However, my intention was to minimize code changes as much as possible.

If you don't mind, I would prefer merging this. Later we can consider a bigger refactor for leveraging `container.run()` with `Detach=False` and `rm=True`.